### PR TITLE
add new GA agents to index/overview docs

### PR DESCRIPTION
**Pull request by jalvz**
depends on: https://github.com/elastic/docs/pull/396

### CODE REVIEW

File: docs/guide/index.asciidoc [hunk 1]
```diff
@@ -251,5 +251,60 @@ pip install elastic-apm
...
+
+Create a config file config/elastic_apm.yml:
+
+[source,bash]
+----------------------------------
+server_url: http://localhost:8100
+secret_token: ''
+----------------------------------
+
+The Ruby agent automatically instruments Rails out of the box.
```
Comment (jalvz) [inline docs/guide/index.asciidoc]: @simitt Rack is also auto-instrumented?

File: docs/guide/index.asciidoc [hunk 2]
```diff
@@ -178,7 +178,7 @@ you need to install an agent in your service.
 === Install and configure APM agents
 
 Agents are written in the same language as your service.
-Currently Elastic APM has agents for Node.js and for Python.
+Currently Elastic APM has agents for Node.js, Python, Ruby, and Javascript.
```
Comment (jahtalab) [inline docs/guide/index.asciidoc]: It should be `JavaScript`

File: docs/guide/index.asciidoc [hunk 3]
```diff
@@ -251,5 +251,60 @@ pip install elastic-apm
...
+
+  // Set custom APM Server URL (default: http://localhost:8200)
+  serverUrl: 'http://localhost:8200',
+
+  // Set service version (required for sourcemap feature)
+  serviceVersion: ''
+})
+----------------------------------
+
+See the {apm-rum-ref}/index.html[APM Javascript RUM Agent documentation] for more details.
```
Comment (jahtalab) [inline docs/guide/index.asciidoc]: Also `RUM JavaScript Agent`

File: docs/overview.asciidoc [hunk 4]
```diff
@@ -28,6 +28,8 @@ please also have a look at the documentation for
 
 * {apm-node-ref}/index.html[APM Node.js Agent]
 * {apm-py-ref}/index.html[APM Python Agent]
+* {apm-ruby-ref}/index.html[APM Ruby Agent]
+* {apm-rum-ref}/index.html[APM Javascript RUM Agent]
```
Comment (jahtalab) [inline docs/overview.asciidoc]: `RUM JavaScript Agent`

File: docs/guide/index.asciidoc (hunk 2 above)
Comment (watson) [inline docs/guide/index.asciidoc]: s/Javascript/JavaScript/

File: docs/guide/index.asciidoc [hunk 5]
```diff
@@ -251,5 +251,60 @@ pip install elastic-apm
...
+secret_token: ''
+----------------------------------
+
+The Ruby agent automatically instruments Rails out of the box.
+See the {apm-ruby-ref}/index.html[APM Ruby Agent documentation] for more details.
+
+
+[[rum-agent]]
+[float]
+==== Install the Javascript RUM agent
```
Comment (watson) [inline docs/guide/index.asciidoc]: s/Javascript/JavaScript/

File: docs/guide/index.asciidoc (hunk 3 above)
Comment (watson) [inline docs/guide/index.asciidoc]: s/Javascript/JavaScript/

File: docs/overview.asciidoc (hunk 4 above)
Comment (watson) [inline docs/overview.asciidoc]: s/Javascript/JavaScript/

File: docs/guide/index.asciidoc [hunk 6]
```diff
@@ -251,5 +251,60 @@ pip install elastic-apm
...
+[source,bash]
+----------------------------------
+gem 'elastic-apm'
+----------------------------------
+
+Create a config file config/elastic_apm.yml:
+
+[source,bash]
+----------------------------------
+server_url: http://localhost:8100
```
Comment (simitt) [inline docs/guide/index.asciidoc]: I think `localhost:8200` would make more sense, as it is the default.

File: docs/guide/index.asciidoc (hunk 1 above)
Comment (simitt) [reply docs/guide/index.asciidoc]: some more setup necessary when not Rails https://www.elastic.co/guide/en/apm/agent/ruby/current/getting-started-rack.html

**Pull request merged**